snap/info_test.go

```diff
@@ -128,6 +128,10 @@ func (s *infoSuite) TestReadInfo(c *C) {
 }
 
 func makeTestSnap(c *C, yaml string) string {
```

Comment:
@mvo5 @kyrofa it's a bit complicated this one had is one makeTestSnap because it needs to make broken ones, which I think the other helper doesn't allow, on the other hand for the hook stuff the suggestion of actually using MakeTestSnapWithFiles is correct